install.sh

```
@@ -0,0 +1,46 @@
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/UniversalLanguageSelector --branch REL1_44
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/UnlinkedWikibase --branch REL1_44
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/WSOAuth --branch REL1_44
+
+
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/Cite --branch REL1_44
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/VisualEditor --branch REL1_44
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/ContentTranslation --branch REL1_44
+git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/WikimediaMessages --branch REL1_44
+
+tfw php8.2 shell
+cd public_html/mediawiki
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)